RN-connect-Database.md

```diff
@@ -0,0 +1,124 @@
+# React Native & Database
```

Comment:
This is really great work adding a database. However, I would say that it might not be entirely relevant to learning the most unique and interesting parts of React Native.

This is your choice but I probably would not spend time showing everyone how to integrate a database. Instead, I would focus specifically on UI development with React Native. You could, for example, spend time building complicated UIs with animations, long lists, etc.